scripts/zones/Selbina/IDs.lua

```diff
@@ -44,7 +44,8 @@ zones[xi.zone.SELBINA] =
     },
     npc =
     {
-        EXPLORER_MOOGLE = 17793131,
+        DEVEAN_DEFAULT                = 124,
```

Comment:
DEVEAN_DEFAULT is a `text` ID. It should be defined in its proper section, above

Reply:
Since this was more confusing than maintainable, I removed it.